docs/src/user_docs/assembly/code_organization.md

```diff
@@ -93,7 +93,17 @@ In the above example we import `std::math::u64` module from the [standard librar
 The set of modules which can be imported by a program can be specified via a Module Provider when instantiating the [Miden Assembler](https://crates.io/crates/miden-assembly) used to compile the program.
 
 ### Constants
-Miden assembly supports constant declarations. These constants are scoped to the module they are defined in and can be used as immediate parameters for Miden assembly instructions. Currently only `push` instruction supports this.
+Miden assembly supports constant declarations. These constants are scoped to the module they are defined in and can be used as immediate parameters for Miden assembly instructions. Constants are supported as immediate values for the following instructions:
+* `push.CONSTANT1.CONSTANT2`
+* `locaddr.CONSTANT`
+* `loc_load.CONSTANT`
+* `loc_loadw.CONSTANT`
+* `loc_store.CONSTANT`
+* `loc_storew.CONSTANT`
+* `mem_load.CONSTANT`
+* `mem_loadw.CONSTANT`
+* `mem_store.CONSTANT`
+* `mem_storew.CONSTANT`
```

Comment:
minor nit: I'm not sure I'd list them all in bullet points - maybe just a simple comma-separated list will do?

Reply:
agreed, will change.